azalea-auth/src/auth.rs

```diff
@@ -335,6 +346,10 @@ pub async fn get_ms_auth_token(
             .json::<AccessTokenResponse>()
             .await
         {
+            if client_id != CLIENT_ID {
+                access_token_response.access_token.insert_str(0, "d=");
+            }
+
```

Comment:
There's other client ids where `d=` shouldn't be added, I think the user should be responsible for adding it themselves (get_minecraft_token already documents it).
```suggestion
```